### PR TITLE
New package: leanify-0.4.3

### PR DESCRIPTION
**Pull request by aajonusonline**

### CODE REVIEW

File: srcpkgs/leanify/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'leanify'
+pkgname=leanify
+version=0.4.3
+revision=1
+wrksrc="Leanify-$version"
+build_style=gnu-makefile
+short_desc="Lightweight lossless file minifier/optimizer"
+maintainer="Aajonus Online <aajonus.online@gmail.com>"
+license="MIT"
+homepage="https://github.com/jayxon/$pkgname"
```
Comment (biopsin) [inline srcpkgs/leanify/template:10]: Please no variabel in homepage, it's cumbersome to do lookups

**Comment by biopsin** [comment]
Please do fix this in your other pkg pulls as well

**Comment by aajonusonline** [comment]
Fixed

**Comment by Chocimier** [comment]
Bundling libraries rather than using shared ones is big disadvantage.

**Comment by aajonusonline** [comment]
Is this a reason not to add this package?

**Comment by aajonusonline** [comment]
is there any progress on this one? it's been a month already.

File: srcpkgs/leanify/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'leanify'
+pkgname=leanify
```
Comment (abenson) [inline srcpkgs/leanify/template:2]: Package name should be `Leanify`, we try to preserve upstream case when possible

**Comment by aajonusonline** [comment]
@abenson is this good?

**Comment by abenson** [comment]
Is there a way to build it using system libraries instead of bundling its own?

File: srcpkgs/Leanify/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'Leanify'
+pkgname=Leanify
+version=0.4.3
+revision=1
+wrksrc="Leanify-$version"
```
Comment (abenson) [inline srcpkgs/Leanify/template:5]: This is the default, you can remove this line

**Comment by abenson** [comment]
Any update on it using system libraries?

**Comment by abenson** [comment]
Seemingly abandoned.

**Pull request closed without merging**